torchaudio/csrc/register.cpp

```diff
@@ -4,21 +4,10 @@
 #include <torchaudio/csrc/sox_effects.h>
 #include <torchaudio/csrc/sox_io.h>
 #include <torchaudio/csrc/sox_utils.h>
-#include <torchaudio/csrc/typedefs.h>
 
 namespace torchaudio {
 namespace {
 
-////////////////////////////////////////////////////////////////////////////////
-// typedefs.h
-////////////////////////////////////////////////////////////////////////////////
-static auto registerSignalInfo =
-    torch::class_<SignalInfo>("torchaudio", "SignalInfo")
-        .def(torch::init<int64_t, int64_t, int64_t>())
```

Comment:
nit: can you elaborate why this line is no longer needed?

Reply:
It is still used and moved to `sox_io.h`. Originally I intended this struct to be somewhat in larger scope, but now it is only used by `get_info` func so I moved it to `torchaudio::sox_io` namespace.